openslides/config/views.py

```diff
@@ -129,12 +129,17 @@ def retrieve(self, request, *args, **kwargs):
             raise Http404
         return Response(data)
 
-    def update(self, request, pk=None):
+    def update(self, request, *args, **kwargs):
         """
-        TODO
+        Updates one config variable. Only managers can do this.
         """
         if not request.user.has_perm('config.can_manage'):
             self.permission_denied(request)
-        else:
-            # TODO: Implement update method
-            self.permission_denied(request)
+        key = kwargs['pk']
+        # Check if pk is a valid config variable key.
+        if key not in config:
+            raise Http404
+        # Change value.
+        config[key] = request.data
```

Comment:
I think we need validation here. This should be possible with the django form fields, as long as we use them